src/sentry/incidents/subscription_processor.py

```diff
@@ -215,8 +215,12 @@ def get_comparison_aggregation_value(
             time_col = ENTITY_TIME_COLUMNS[get_entity_key_from_query_builder(query_builder)]
             query_builder.add_conditions(
                 [
-                    Condition(Column(time_col), Op.GTE, start),
-                    Condition(Column(time_col), Op.LT, end),
+                    Condition(
+                        Column(time_col, entity=Entity("events", alias="events")), Op.GTE, start
+                    ),
+                    Condition(
+                        Column(time_col, entity=Entity("events", alias="events")), Op.LT, end
```

Comment:
are you sure this is the right entity? is this function strictly for the events? I see other functions like "get entity key from..." which make me think it might not always be the same

Reply:
@ceorourke would you know?

Reply:
I do not, the only time I used this I knew which entity I was using for sure.